SmartSearch.Abstractions/IField.cs

```diff
@@ -24,6 +24,7 @@ public interface IField
         string Name { get; }
 
         FieldRelevance Relevance { get; }
+        float RelevanceBoost { get; }
```

Comment:
Brings the Relevance Boost value closer to the Relevance definition for the field.